cookbooks/bcpc/templates/default/pam_namespace.init.erb

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Please pull in comment https://github.com/bloomberg/chef-bach/pull/1278/#issuecomment-427468873 to the script so it persists with the file -- and we remove this file on upgrade of Ubuntu.